modules/installation-adding-registry-pull-secret.adoc

```diff
@@ -205,3 +212,8 @@ ifeval::["{context}" == "installing-mirroring-disconnected"]
 :!restricted:
 :!oc-mirror:
 endif::[]
+
+ifeval::["{context}" == "about-installing-oc-mirror-v2"]
+:restricted:
```

Comment:
These are missing the "!" syntax that undefines the attributes. Look above for examples.